src/Apps/Collect/Components/Collection/Header/index.tsx

```diff
@@ -248,10 +253,13 @@ export const CollectionHeader: FC<Props> = ({ artworks, collection }) => {
         return (
           <header>
             <Flex flexDirection="column">
-              <Box>
+              <Box mt={[0, "-12px"]}>
```

Comment:
This is ugly, and not in keeping with the spirit of the design system. But it gets rid of the gap that would otherwise appear here:

![gap](https://user-images.githubusercontent.com/140521/61672234-df30fb80-acb8-11e9-8764-429e70dcda05.png)

There is probably a way to change the styling of the `main-layout-container` in Force to accomplish this as well, but I looked into that and grew worried about ripple effects. Happy to pair on that though if anyone thinks we should.